specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-10-01/examples/WafListManagedRuleSets.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
For new API versions, please copy a prior reviewed API version as the first commit, and then commit the changes. It makes it easier to review deltas in Github natively. Thanks

Reply:
The first commit should be a copy of the 2019-03-01 API. The subsequent commits have the proposed changes for version 2019-10-01.